frontend/src/app/modules/fields/edit/field-types/select-edit-field.component.ts

```diff
@@ -138,6 +138,15 @@ export class SelectEditFieldComponent extends EditFieldComponent implements OnIn
     return Promise.resolve();
   }
 
+  private get allowedValuesParams() {
+    const type = this.changeset.value('type');
+    if (this.name === 'project' && type && type.id) {
+      return { for_type: type.id };
+    }
+
+    return {};
```

Comment:
I believe this is the wrong way to address the problem. The filter should instead be made part of the available_projects link in the form/schema so that the front end can remain unknowledgeable. 

Possibly unrelated: The available_projects url gets called twice when using the global full screen create form:
![image](https://user-images.githubusercontent.com/617519/58783865-4d730f00-85e2-11e9-9597-f41cb19c2eb8.png)

And the wp form get's also called twice (again, I believe we fixed that once) when using the inline create inside an embedded table:
![image](https://user-images.githubusercontent.com/617519/58783991-9925b880-85e2-11e9-847f-728e56d03a51.png)

Reply:
Double projects loading is caused by version completer. They don't share the same resource so this gets loaded twice. (allowedValue link vs fixed path request)

Reply:
The form is loaded twice because of the filter:

- Once it's loaded for the initial creation
- Then the type is set from filters, requiring a form reload